kadena/light-client/src/bin/client.rs

```diff
@@ -1,17 +1,29 @@
 // Copyright (c) Argument Computer Corporation
 // SPDX-License-Identifier: Apache-2.0
 
-use anyhow::Result;
+use anyhow::{anyhow, Result};
 use clap::Parser;
+use getset::Getters;
+use kadena_lc::client::error::ClientError;
 use kadena_lc::client::Client;
-use kadena_lc::proofs::ProvingMode;
+use kadena_lc::proofs::longest_chain::LongestChainOut;
+use kadena_lc::proofs::spv::SpvOut;
+use kadena_lc::proofs::{ProofType, ProvingMode};
 use kadena_lc_core::crypto::hash::HashValue;
-use kadena_lc_core::types::header::layer::ChainwebLayerHeader;
+use kadena_lc_core::crypto::U256;
+use kadena_lc_core::merkle::spv::Spv;
+use log::{debug, error, info};
 use std::env;
+use std::fmt::Display;
 use std::sync::Arc;
+use std::time::Duration;
+use tokio::sync::{mpsc, OwnedSemaphorePermit, RwLock, Semaphore};
+use tokio::task::JoinHandle;
 
-pub const TARGET_BLOCK: usize = 5099345;
+pub const CHECKPOINT_BLOCK_HEIGHT: usize = 5158073;
 pub const BLOCK_WINDOW: usize = 3;
+pub const VERIFIER_STORED_CHECKPOINT_COUNT: usize = 5;
+pub const CONFIRMATION_WORK_THRESHOLD: usize = 0;
```

Comment:
Is this expected to be `0` in this PR, and will be updated to another value later on?

Reply:
I will update it when I get an idea of the actual threshold that would make sense for KDA. I have currently no idea.